### ISSUE TITLE
Need http2 on the server

### ISSUE THREAD

**Issue by dotnetCarpenter** [Firefund/styleguide#100]
Need http2 on the server: https://www.npmjs.com/package/http2

Important for our load strategy outlined in #99.

**Comment by carbureted** [comment]
Support for http2 with express was pretty janky last time I tried it, are
we sure this isn't premature optimization?

**Comment by dotnetCarpenter** [comment]
yes, we are not going to bundle the majority of css and we will lazy load
some html and js as web components.
In short, Web Components does not make sense without http2.
Which module did you try? This one: https://www.npmjs.com/package/http2 ??

**Comment by carbureted** [comment]
Yeah, for instance see https://github.com/molnarg/node-http2/issues/100 or https://github.com/expressjs/express/issues/2761. Last time I used it there were some hacks that would get a basic express app working, but it definitely didn't feel stable (other modules would occasionally crash, etc). Things might have changed in past months, though.
